ci-operator/step-registry/cloud-maintenance/aws/s3bucket-cleanup/cloud-maintenance-aws-s3bucket-cleanup-commands.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Set varaibles needed to login to AWS
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+export AWS_CONFIG_FILE=$CLUSTER_PROFILE_DIR/.aws
+
+KEY_ID=$(cat $AWS_SHARED_CREDENTIALS_FILE | grep aws_access_key_id | tr -d ' ' | cut -d '=' -f 2)
+ACCESS_KEY=$(cat $AWS_SHARED_CREDENTIALS_FILE | grep aws_secret_access_key | tr -d ' ' | cut -d '=' -f 2)
+export AWS_ACCESS_KEY_ID=$KEY_ID
+export AWS_SECRET_ACCESS_KEY=$ACCESS_KEY
+
+# Remove any spaces from the exclude list
+EXCLUDE_LIST=$(echo "$EXCLUDE_LIST" | tr -d ' ')
+
+# Convert the exclude list to an array using a comma as the delimiter
+exclude_array=()
+if [ -n "$EXCLUDE_LIST" ]; then
+    IFS=',' read -ra exclude_array <<< "$EXCLUDE_LIST"
```

Comment:
what the `IFS` variable?

Reply:
@Simranpal`IFS`is the variable used to determine the separator when parsing the `EXCLUDE_LIST` variable. In this case it used the comma value to determine a new item in the list of buckets to exclude.

Here is some documentation that explains this variable more: https://bash.cyberciti.biz/guide/$IFS